a11y note about label & placeholder, minor text revisions (#1837)
carbon-design-system/carbon-website: src/content/components/search/usage.mdx

---
title: Search
tabs: ['Code', 'Usage', 'Style']
---

## General guidance

> _Search_ enables users to specify a word or a phrase to find particular relevant pieces of content without the use of navigation. Search can be used as the primary means of discovering content, or as a filter to aid the user in finding content.

Reviewer: ```suggestion
Search enables users to specify a word or a phrase to find relevant pieces of content without the use of navigation. Search can be used as the primary means of discovering content, or as a filter to aid the user in finding content.
```
Removing italics

## Variations

| Search type    | Purpose                                                                                                                                                                                              |
| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | _Large search_ | This should be used at a global level, when the user is searching content within a page view.                                                                                                        |

Reviewer: ```suggestion
| Large search | This should be used at a global level when the user is searching content within a page view.                                                                                                        |
```

> | _Small search_ | Choose small when there are space constraints in your design. It can also be component specific. For example, small search can be used to filter data within a [data table](/components/data-table). |

Reviewer: ```suggestion
| Small search | Choose small when there are space constraints in your design. It can also be component-specific. For example, small search can be used to filter data within a [data table](/components/data-table). |
```

> Set users' context for the search with helpful placeholder text within the search field (search documents), that pertains to the page or section the search box is in. Search in the global header should just say search.

Reviewer: ```suggestion
Set the user's context for the search with helpful placeholder text within the search field (search documents), that pertains to the page or section the search box is in. Search in the global header should just say search.
```

> Accessibility note: If the search component's label and placeholder text differ, VoiceOver on Mac OSX will read both.

Reviewer: ```suggestion
Accessibility note: If the search component's label and placeholder text differ, voiceover on macOS will read both.
```

Author: I accepted the change for "macOS" but reverted "VoiceOver" as that is the program's official name
